css/_base.scss

```diff
@@ -51,8 +51,10 @@ body {
     }
 }
 
-.jitsi-icon svg {
```

Comment:
this was preventing colors provided as props to be applied, since it had priority over them

Reply:
Does this change not break anything?

Reply:
the result is that any icon that had a fill override will not get the default fill from css applied, so nothing should break